hooks/useConditionalTokens.ts

```diff
@@ -47,53 +48,53 @@ export default function useConditionalTokens() {
     fetchAmount: fetchUnderlyingMeta,
   } = useBalance(base.underlyingTokenMint);
   const {
-      amount: pMetaBalance,
-      fetchAmount: fetchPassMeta,
+    amount: pMetaBalance,
+    fetchAmount: fetchPassMeta,
   } = useBalance(base.conditionalOnFinalizeTokenMint);
```

Comment:
These are the things I believe we should fetch initially, understand if the accounts exist, if they do, shove them into an array and do an accountSubscribe in a global websocket. Then wrap every time we'd want to fetch with a check to ensure WS is connected and subscription exists if not then do a manual fetch. Does that make sense?

Reply:
@R-K-H yeah that totally makes sense. Basically wallet token balances connected with WS instead of continual fetching, and then wrap that in fallback for polling mechanism if WS fails.